src/app/app.module.ts

```diff
@@ -14,7 +14,14 @@ import { MatButtonModule, MatCheckboxModule} from '@angular/material';
   imports: [
     BrowserModule,
     BrowserAnimationsModule,
-    FeCommonModule
+    FeCommonModule,
+    MatToolbarModule
+  ],
+  exports: [
+    BrowserModule,
+    BrowserAnimationsModule,
+    FeCommonModule,
+    MatToolbarModule
```

Comment:
Is MatToolBarModule required here? These can get out of controller quickly so we want to limit how much is included in the master scope.

Reply:
Nope I missed that one, I'll remove it